### ISSUE TITLE
make check failures but make install worked

### ISSUE THREAD

**Issue by zacmon** [qmarcou/IGoR#61]
Hi,

I installed Igor 1-4 what appears to be successfully: `make` and `make install` presented no issues that broke either step. I used gcc 6.3.1 20170216 on an HPC running CentOS Linux release 7.6.1810. However, I received this `make check` message which cause the `make check` to break. Is this something I should be concerned about while running igor?

```
Making check in cdf
make[3]: Entering directory '/gscratch/stf/zachmon/software/igor_1-4-0/libs/gsl_sub/cdf'
make  test
make[4]: Entering directory '/gscratch/stf/zachmon/software/igor_1-4-0/libs/gsl_sub/cdf'
make[4]: *** No rule to make target '../randist/libgslrandist.la', needed by 'test'.  Stop.
make[4]: Leaving directory '/gscratch/stf/zachmon/software/igor_1-4-0/libs/gsl_sub/cdf'
Makefile:966: recipe for target 'check-am' failed
make[3]: *** [check-am] Error 2
make[3]: Leaving directory '/gscratch/stf/zachmon/software/igor_1-4-0/libs/gsl_sub/cdf'
Makefile:790: recipe for target 'check-recursive' failed
make[2]: *** [check-recursive] Error 1
make[2]: Leaving directory '/gscratch/stf/zachmon/software/igor_1-4-0/libs/gsl_sub'
Makefile:357: recipe for target 'check-recursive' failed
make[1]: *** [check-recursive] Error 1
make[1]: Leaving directory '/gscratch/stf/zachmon/software/igor_1-4-0/libs'
Makefile:553: recipe for target 'check-recursive' failed
make: *** [check-recursive] Error 1
```

Thanks,
Zach

**Comment by qmarcou** [comment]
Dear @zacmon 
This answer is most likely given way to late to still be of interest to you, my apologies
I did not implement checks for the igor source part (which is bad), and the checks that are run come from the GSL subparts that I had stripped from a lot of components that were not necessary for IGoR to run. This was a (not necessarily good) compromise between shipping all necessary code along with IGoR and keeping a more lightweight program (e.g in terms of compile time).

In the process of stripping GSL parts I might not have paid attention to not breaking GSL tests, which is why you get this error.

So in a nutshell, no you shouldn't be concerned by this

best regards
